exploration/test_grid_alignment.md

```diff
@@ -0,0 +1,100 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.3
+  kernelspec:
+    display_name: venv
+    language: python
+    name: python3
+---
+
+## Grid alignment investigation
+
+This notebook investigates the source of some grid misalignment identified during preparation of the source raster data. We're looking at a sample SEAS5 COG and demonstrating how the order of clipping and resampling operations can impact the output raster stats.
+
+```python
+from src.utils.raster_utils import upsample_raster
+from src.utils.cog_utils import get_cog_url
+
+import rioxarray as rxr
+import geopandas as gpd
+from rasterstats import zonal_stats
+import numpy as np
+import pandas as pd
+
+MODE = "prod"
+date = "2024-01-01"
+```
+
+#### 1. Read in source raster and geopandas dataframe
+
+```python
+# Or replace with other locally saved shapefile
+gdf = gpd.read_file("data/eth_adm2_simplified.shp")
```

Comment:
can we put this on blob? or if you want to share data folder we can also do that